modules/extensions/opagatekeeper.tf

```diff
@@ -0,0 +1,33 @@
+# Copyright 2017, 2021 Oracle Corporation and/or affiliates.
```

Comment:
Was this file really created in 2017? If not, you should only list 2021.

Reply:
Changed to 2021